tools/astarte_import/lib/astarte/import/cli.ex

```diff
@@ -17,13 +17,16 @@
 #
 
 defmodule Astarte.Import.CLI do
+  alias Astarte.Import.Cluster
   alias Astarte.Import.PopulateDB
   require Logger
 
   @chunk_size 4096
 
   def main(args) do
     with {:started, {:ok, _}} <- {:started, Application.ensure_all_started(:astarte_import)},
+         {:cluster_registry, :ok} <-
+           {:cluster_registry, Cluster.ensure_registered()},
```

Comment:
I believe `Cluster.ensure_registered` can only return `:ok`
if so, we can just

```suggestion
         :ok = Cluster.ensure_registered(),
```

so that when reading the code we know it can't fail